File: memory-domain/src/main/java/com/memory/domain/hashtag/repository/MemoryHashTagRepositoryCustom.java
package com.memory.domain.hashtag.repository;

import com.memory.domain.hashtag.MemoryHashTag;
import com.memory.domain.memory.Memory;

import java.util.List;

public interface MemoryHashTagRepositoryCustom {
    List<MemoryHashTag> findByMemory(Memory memory);
    void deleteByMemory(Memory memory);
}

File: memory-domain/src/main/kotlin/com/memory/domain/hashtag/repository/MemoryHashTagRepositoryCustom.kt
package com.memory.domain.hashtag.repository

import com.memory.domain.hashtag.MemoryHashTag
import com.memory.domain.memory.Memory

interface MemoryHashTagRepositoryCustom {
    fun findByMemory(memory: Memory?): List<MemoryHashTag>
    fun deleteByMemory(memory: Memory?)
}
